doc/Command-Reference.md

```diff
@@ -9946,8 +9947,55 @@ In SONiC, there usually exists a set of tables related/relevant to a particular
 	    }
 	}	
   ```
+
+### Event Driven Techsupport Invocation
+
+This feature/capability makes the techsupport invocation event-driven based on core dump generation. This feature is only applicable for the processes running in the containers. More detailed explanation can be found in the HLD https://github.com/Azure/SONiC/blob/master/doc/auto_techsupport_and_coredump_mgmt.md
+
+- Usage:
+  ```
+  1) Configuration:
+  config auto-techsupport global state <enabled/disabled>
+  config auto-techsupport global rate-limit-interval <uint16>
+  config auto-techsupport global max-techsupport-limit <float upto two decimal places>
+  config auto-techsupport global max-core-limit <float upto two decimal places>
+  config auto-techsupport global since <string>
   
+  config auto-techsupport-feature add <feature_name> --state <enabled/disabled> --rate-limit-interval <uint16>
+  config auto-techsupport-feature delete <feature_name>
+  config auto-techsupport-feature update <feature_name> --state <enabled/disabled> 
+  config auto-techsupport-feature update <feature_name> --rate-limit-interval <uint16>
 		
+  2) Show CLI:
+  show auto-techsupport global
+  show auto-techsupport-feature 
```

Comment:
is - is correct for feature?

Reply:
yes it is. Eg: https://github.com/Azure/SONiC/blob/master/doc/auto_techsupport_and_coredump_mgmt.md#show-cli